libraries/SITL/SIM_VisualOdometry.h

```diff
@@ -0,0 +1,79 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  simple particle sensor simulation
```

Comment:
Nice to see this incorrect comment has been faithfully propagate :-)

Reply:
I have added a test that is similar to the current VisionPosition test in operation but differs in terms of parameters. Also fixed the incorrect comment above and reordered the commits.

Reply:
@peterbarker it seems the CI build encountered some errors. Sorry for the novice question, but can you guide me on how to fix the problem? I am new to the CI pipeline.

Reply:
I've kicked over the failing tests.  Azure is just like that.